funcx_endpoint/tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py{310,39,38,37,36}
+skip_missing_interpreters = true
+
+[testenv]
+usedevelop = true
+extras = test
+commands =
+    safety check
```

Comment:
If `safety check` is going to run under tox, it should have a dedicated testenv, not be part of the main one.
As an example, here's how I did this for `mypy` in `funcx-common`:
https://github.com/funcx-faas/funcx-common/blob/c5092367bbc3f048066c987d4e2b98fa2e485ec8/tox.ini#L39-L42
(that allows `tox -e mypy`)

Otherwise, if I `cd funcx_endpoint; tox` in order to test on all of my installed pythons, I'm going to be running `safety check` once per python version, when what I really want is to run the tests.